packages/functional-tests/tests/react-conversion/syncV3ResetPassword.spec.ts

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, newPagesForSync } from '../../lib/fixtures/standard';
+import { EmailHeader, EmailType } from '../../lib/email';
+import { getReactFeatureFlagUrl } from '../../lib/react-flag';
+
+test.describe.configure({ mode: 'parallel' });
+
+test.describe('Firefox Desktop Sync v3 reset password react', () => {
+  test.beforeEach(async ({ pages: { login } }) => {
+    test.slow();
+    // Ensure that the feature flag is enabled
+    const config = await login.getConfig();
+    test.skip(config.showReactApp.resetPasswordRoutes !== true);
```

Comment:
Suprised this didn't fail but I think it could become flaky. I think it is prefered to do something like this in the beforeEach and afterEach, ref: https://github.com/mozilla/fxa/blob/880d9c59a774fa57a8e730b7a5cbb5b4c5bd994f/packages/functional-tests/tests/oauth/syncSignIn.spec.ts#L11